pkg/apply/prune/prune.go

```diff
@@ -96,6 +100,17 @@ func (po *PruneOptions) Prune(localInv *unstructured.Unstructured, localObjs []*
 	for _, clusterObj := range clusterObjs {
 		mapping, err := po.mapper.RESTMapping(clusterObj.GroupKind)
 		if err != nil {
+			u := &unstructured.Unstructured{}
+			u.SetGroupVersionKind(schema.GroupVersionKind{
+				Group: clusterObj.GroupKind.Group,
+				Kind:  clusterObj.GroupKind.Kind,
+			})
+			u.SetNamespace(clusterObj.Namespace)
+			u.SetName((clusterObj.Name))
+			eventChannel <- createErrorEvent(u, err)
```

Comment:
I think we should avoid creating unstructured objects that aren't actually either complete resources read from a manifest or resources read from the cluster. We do have the `object.ObjMeta` type for capturing the identifying information about a resource.

Reply:
Ack, I think adding the error in the `PruneEvent` will remove this change.